kaylaewood/synonyms-sugar
# Add functionality to WordBox and buttons

#### What's this PR do?
This PR adds functionality to clicking the buttons in the `WordBox` component. Now, you can fetch synonyms from the new words by clicking on the synonyms.
#### Where should the reviewer start?
Start by checking out the `WordBox.svelte` file.
#### How should this be manually tested?
Try it out in the browser! You should be able to type in a word, click `Go!`, and then click on any synonyms.
